docs/admin/admin_install.md

```diff
@@ -55,6 +55,7 @@ PLUGINS_CONFIG = {
         "enable_intended": True,
         "enable_sotagg": True,
         "sot_agg_transposer": None,
+        "enable_postprocessing": True,
```

Comment:
Shouldn't `config_postprocessing_callables` be here?

Reply:
resolved in 5b88c8a

Reply:
I think we should have in docs a default secure set up, for the copy and paste.

Reply:
```suggestion
        "enable_postprocessing": False,
```